### DISCUSSION TITLE
Error after installation

### DISCUSSION THREAD

**Discussion by jurd** [GetPublii/Publii#1597] [General]
Hi, I've been using Publii for a while now. At home on win11 and at work on a win 8.1 (64 bit) Server. After the last update to Publii-0.45.2 on win8.1 the program stopped starting. Even when I installed the older version of Publii-0.44.1, I still have the same problem.

![publiierror](https://github.com/user-attachments/assets/5e4f9fe8-bb8c-46e4-948e-23fe31f583c5)

Any ideas what to do with this?
regards
jurand

**Comment by dziudek** [comment]
Hi @jurd ,

Electron for a longer time not support Windows 7/8/8.1 so Publii too: https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice

**Comment by jurd** [comment]
Thanks for the answer. 
Too bad. Is there still an old version of Publii available that works on Win 8.1?

**Reply by dziudek** [reply]
@jurd - probably v.0.42.1 should work as it uses Electron 22.*: https://getpublii.com/download/Publii-0.42.1.exe

**Reply by jurd** [reply]
Thanks, it works!
